src/app/services/documentos.service.ts

```diff
@@ -38,6 +42,26 @@ export class DocumentosService {
         return this.http.post(this.pdfURL + '/pdf', data, options)
             .map(this.extractData)
             .catch(this.handleError);
+
+    }
+
+    descargarArchivo(informe, nombreArchivo: string, headers: any): Promise<any> {
+        return new Promise((resolve, reject) => {
+            this.descargarV2(informe).subscribe(data => {
```

Comment:
en vez de usar promesas usar el operador tap de RXJS. 
```suggestion
            this.descargarV2(informe).pipe( tap( data => { }) 
```

Y en prestacionValidación hacer: 

```javascript
this.servicioDocumentos.descargarArchivo(informe, term, { type: 'application/pdf' }).subscribe(() => {
   this.descargaCerrada = false;
})
```

Reply:
Bien pensado, ya quedó 2